tracer/test/snapshots/OpenTelemetrySdkTests_1_0.verified.txt

```diff
@@ -163,7 +187,7 @@
     SpanId: Id_10,
     Name: OtherLibrary.internal,
     Resource: Response,
-    Service: OtherLibrary,
+    Service: MyServiceName,
```

Comment:
After rebasing, this snapshot was added. This is also intentional and called out in the PR description. And since this is no longer a top-level span, the corresponding metric is removed